### DISCUSSION TITLE
What is NvimTreeExecFile?

### DISCUSSION THREAD

**Discussion by cheeze2000** [nvim-tree/nvim-tree.lua#1623] [Q&A] [answered]
What is this highlight group? All my `.js` files have this highlight applied to them so now the names are bolded and green. I couldn't find any documentation regarding what this is.

**Comment by gegoune** [comment, marked as answer]
It highlights files with executable bit:

https://github.com/kyazdani42/nvim-tree.lua/blob/7282f7de8aedf861fe0162a559fc2b214383c51c/lua/nvim-tree/renderer/builder.lua#L219-L220

**Reply by cheeze2000** [reply]
![image](https://user-images.githubusercontent.com/26966445/193744245-cf1381cf-e5ab-4487-ab3f-3916aac3e687.png)
These files are highlighted with NvimTreeExecFile, I verified it by modifying highlight group to `guifg=blue` and the text became blue.

![image](https://user-images.githubusercontent.com/26966445/193744386-881ef288-57a8-4625-957b-1a7f2ad76578.png)
The files are not executable.

**Reply by alex-courtis** [reply]
I'm not able to replicate this one.

Which operating system are you using?

Please raise a [bug](https://github.com/kyazdani42/nvim-tree.lua/issues/new?assignees=&labels=bug&template=bug_report.yml)

**Reply by cheeze2000** [reply]
> I'm not able to replicate this one.
> 
> Which operating system are you using?
> 
> Please raise a [bug](https://github.com/kyazdani42/nvim-tree.lua/issues/new?assignees=&labels=bug&template=bug_report.yml)

Hello, I have recently switched to using an Arch Linux VM from Ubuntu WSL2. I can't seem to reproduce this issue too. The .js files are not highlighted anymore.

**Reply by alex-courtis** [reply]
> Hello, I have recently switched to using an Arch Linux VM from Ubuntu WSL2. I can't seem to reproduce this issue too. The .js files are not highlighted anymore.

Why not just install Arch Linux? Much simpler than windows :)

You might follow my [simple installation recipe](https://github.com/alex-courtis/arch/blob/master/doc/arch-install.md).
